docs/wiki/dev-guides/Release-Process.md

```diff
@@ -249,6 +249,27 @@ View at:
 https://pypi.org/project/csp/<version number>/
 ```
 
+### Releasing to conda-forge
```

Comment:
I don't think this is sufficient or necessary, there are lots of detailed docs in the Conda Forge docs, can just point to those.

Reply:
I'll defer to @robambalu who asked for this.